tethys_portal/static/tethys_portal/js/sso_tenant.js

```diff
@@ -0,0 +1,43 @@
+$(function() {
+  if (window.localStorage) {
+    // Show the hidden remember checkbox only if local storage is supported
+    $('label[for="id_remember"]').css('display', 'block');
+
+    // Check local storage for saved data
+    let backend = $('#backend').val();
+    let remember_key = `${backend}-remember`;
+    let tenant_key = `${backend}-tenant`;
+    let remember = localStorage.getItem(remember_key);
+    let tenant = localStorage.getItem(tenant_key);
+    console.log(backend);
+    console.log(remember_key);
+    console.log(tenant_key);
```

Comment:
Is there a reason to leave these logs in the live version?

Reply:
Nope. Just removed them.